components/bio-formats-tools/src/loci/formats/tools/ImageConverter.java

```diff
@@ -384,6 +388,10 @@ private void printUsage() {
       "      -no-sequential: do not assume that planes are written in sequential order",
       "               -swap: override the default input dimension order; argument is f.i. XYCTZ",
       "               -fill: byte value to use for undefined pixels (0-255)",
+      "      -precompressed: transfer compressed bytes from input dataset directly to output.",
+      "                      Most input and output formats do not support this option.",
```

Comment:
Two quick brainstormy thoughts:
 * I could see an option to `formatlist` that would print out a table of which of these interfaces is supported by each format.
 * It would be interesting to see (OME-)TIFF as another supported writer format for this to have an N>2.

Reply:
Agree these would be nice targets for future work, but are well outside the scope of this PR and not realistic for 7.1.0. OK to open new issues for each, so that we can schedule for future releases?